Source/DafnyCore/Rewriters/RefinementTransformer.cs

```diff
@@ -58,7 +58,8 @@ public RefinementToken(IToken tok, ModuleDefinition m)
       }
 
       var rtok = tok as RefinementToken;
-      return rtok != null && rtok.InheritingModule == m;
+      var result = rtok != null && rtok.InheritingModule == m;
+      return result;
```

Comment:
Non-blocking: this refactoring introduces a variable that does not seem to provide meaning to the result